src/sbt-test/test-output/test-html-output/test

```diff
@@ -1,4 +1,4 @@
-# Test is an output report is created
-> findSecBugs
+# Tests if an output report is created
+-> findSecBugs
```

Comment:
According to [the documentation of scripted], `>` means 'run the plugin and expect it to succeed, while `->` means 'run the plugin and expect it to fail'. Yay for symbolic methods, I guess?